src/cli-options.ts

```diff
@@ -280,6 +280,12 @@ const cliOptions: CLIOption[] = [
     default: false,
     type: 'boolean',
   },
+  {
+    long: 'cacheClear',
+    description: 'Remove the default cache file',
+    default: false,
```

Comment:
Let's omit the `default` property since it is assumed to be false when unspecified.